packages/utils-dom/tests/axe.ts

```diff
@@ -71,6 +93,13 @@ test.serial('aria-hidden-focus', async (t) => {
     });
 });
 
+test.serial('form-field-multiple-labels', async (t) => {
+    await testAxe(t, {
+        incomplete: '<label for="test">Hi</label><label for="test">Foo</label><input type="text" id="test" />',
```

Comment:
Generally, an incomplete rule suggests a bug. Is there a reason we expect this case to be incomplete versus explicitly passing or failing?

Reply:
As axe-core is reporting this as incomplete I created some logic to pass if the right 'incomplete' was reported (per offline discussion)